scripts/calendar/hindu_generator.py

```
@@ -0,0 +1,821 @@
+
+_LAHIRI_J2000 = 23.85045  # degrees at J2000.0 (JD 2451545.0)
+_PRECESSION_RATE = 50.2388475 / 3600  # degrees per Julian year
+
+
+class _Astronomy:
+    """Astronimcal helper functions for Hindu calendar calculations."""
+
+    _sun = ephem.Sun()
+    _moon = ephem.Moon()
+
+    def __init__(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)